src/commands/Matcha.ts

```diff
@@ -181,6 +198,17 @@ export default class Matcha extends Command {
           await super.edit(interaction, { content: '/matcha was canceled!', components: [] });
           return;
         }
+        const groupsize = interaction.options.getInteger('groupsize', true);
+
+        if (groupsize <= 1) {
+          await super.edit(interaction, {
+            content: 'You need to enter an integer greater than 1 otherwise your only friend will be yourself :angry:',
+            components: [],
+            ephemeral: true,
+          });
+          return;
+        }
```

Comment:
Nit: I think we can rewrite this error message to be more in line with what our other messages look like

Instead of integer, we can say "group size", and also add some punctuation

`**/matcha** needs a group size greater than 1! Otherwise, your only friend will be yourself! :rage:`